docs/DeveloperGuide.adoc

```diff
@@ -1194,25 +1227,50 @@ Use case resumes at step 2.
 +
 Use case ends.
 
+*Extensions*
+* 2a. Lead entered has a phone number or email address stored earlier.
+** 2a1. CRM Book displays message that a duplicate is detected.
++
+Use case ends.
+
+* 2b. Lead entered does not have a phone number or email address stored earlier.
+** 2b1. CRM Book adds the Lead.
+** 2b2. CRM Book displays message that the Lead has been added.
```

Comment:
MSS

Reply:
don't understand why it's still there after I removed it.
I think it's ok now